_includes/scripts.html

```diff
@@ -1,10 +1,6 @@
-{% assign _scripts = '' | split: '' %}
-{% assign _scripts = _scripts
-  | push: site.scripts
-  | push: layout.scripts
-  | push: page.scripts
-  | uniq %}
-{% for _list in _scripts %}{% for _script in _list %}
-  {% assign _src = _script.src | default: _script %}
+{% assign _scripts = '' | split: '' %} {% assign _scripts = _scripts | push:
+site.scripts | push: layout.scripts | push: page.scripts | uniq %} {% for _list
+in _scripts %}{% for _script in _list %} {% assign _src = _script.src | default:
+_script %}
```

Comment:
I think the old formatting with the newlines for the liquid directives was much easier to read, WDYT about bringing it back?